adr/adr-template.md

```diff
@@ -0,0 +1,17 @@
+# ADR <Four digit number>: <Title>
+
+## Status
+
+The status of this ADR. It can be either Proposed, Accepted, Rejected, Deprecated, Superseded.
```

Comment:
I _assume_ that any existing ADR could be deprecated/superseded by the outcome of a new discussion thread but I think it is still worth explaining `Deprecated` and `Superseded` including the differences between these two, or rather the process that leads to these statuses, should be explained.

If we propose changes via a separate discussion thread and an ADR is only written after that discussion has ended, we should never have an ADR in the status `Proposed`.

Similarly, what is the purpose of documenting something as `Rejected`? If a discussion leads to something else than the originally suggested design, can't we just record this as a different design being `Accepted`?

Explanations for these terms should probably be added to adr-process.md as necessary.